core/Utils.py

```diff
@@ -94,3 +96,37 @@ def print_yaml_nicely(cls, to_print):
         """
         import json
         print json.dumps(to_print, indent=2)
+
+"""
+Used for re-typing variables
```

Comment:
I understand what your are trying to do here. But the fact is that all languages will have this kind of problem. In English "2" can be returned as "to", "two" or "too". In other language I suppose it's the same. And it would be very hard to handle each operation exception for each supported language in kalliope.
So, from my point of view, your neuron is very well coded but cannot be integrated as it as it doesn't take in charge every language.